website/dashboard/test_plots.js

```diff
@@ -0,0 +1,58 @@
+document.addEventListener('DOMContentLoaded', function() {
+    fetch('report-summary.json')
+        .then(response => response.json())
+        .then(data => {
+            function createPlot(container, parameter) {
+                var reference_values = data.map(d => d[parameter]);
+                var fit_values = data.map(d => d[parameter + '_fit']);
+                var errors = fit_values.map((d, i) => d - reference_values[i]);
+
+                var tolerance = reference_values.map((d, i) => data[i]['atol'][parameter] + data[i]['rtol'][parameter] * d);
```

Comment:
This doesn't seem right to me, both the code and the plots. For example, your last plot shows the error bounds decreasing as x increases, that could only happen if `rtol<0`.

In the code you calculate the tolerance values for every single point then calculate the min and max x-value, then plot the first 2 tolerances with those min and max values? Shouldn't it be the other way around? Calculate the min and max x-values and then calculate the y tolerances?

Reply:
Just pushed the changes, here is the last update:
<img width="391" alt="image" src="https://github.com/user-attachments/assets/5c4b6e38-65a3-4488-b277-e425a446e95e">